rpc/src/rpc.rs

```diff
@@ -610,12 +610,20 @@ impl JsonRpcRequestProcessor {
         // epoch
         let bank = self.get_bank_with_config(context_config)?;
 
-        // DO NOT CLEAN UP with feature_set::enable_partitioned_epoch_reward
```

Comment:
should we retain both feature gate names in this comment for clarity?

Reply:
I don't have a strong opinion. I was thinking we would remove the other feature entirely when it is rolled back, which is why I just replaced the name here. But including both and either updating this comment when the feature is removed or leaving the name for historical reference would be fine with me.

Reply:
As per the thread below, I will readdress this RPC area in a follow-up.